ci/fix_wheels.py

```diff
@@ -0,0 +1,55 @@
+import os
+import shutil
+import struct
+import sys
+import zipfile
+
+try:
+    _, wheel_path, dest_dir = sys.argv
+    # Figure out whether we are building on 32 or 64 bit python
+    is_32 = struct.calcsize("P") * 8 == 32
```

Comment:
The standard library has documentation on how to test this. Probably worth following that

https://docs.python.org/3/library/platform.html#platform.architecture